test/v1/connectors/DeleteCodingOutConnectorSpec.scala

```diff
@@ -17,58 +17,57 @@
 package v1.connectors
 
 import api.connectors.ConnectorSpec
-import api.mocks.MockHttpClient
-import mocks.MockAppConfig
-import uk.gov.hmrc.http.HeaderCarrier
-import api.models.domain.Nino
+import api.models.domain.{Nino, TaxYear}
 import api.models.outcomes.ResponseWrapper
 import v1.models.request.deleteCodingOut.DeleteCodingOutParsedRequest
 
 import scala.concurrent.Future
 
 class DeleteCodingOutConnectorSpec extends ConnectorSpec {
 
-  val nino: String    = "AA111111A"
-  val taxYear: String = "2021-22"
+  "DeleteCodingOutConnector" should {
+    "return a 204 status for a success scenario" when {
+      "a valid non-TYS request is made" in new Ifs1Test with Test {
+        def taxYear: TaxYear = TaxYear.fromMtd("2019-20")
 
-  val request: DeleteCodingOutParsedRequest = DeleteCodingOutParsedRequest(
-    nino = Nino(nino),
-    taxYear = taxYear
-  )
+        val outcome = Right(ResponseWrapper(correlationId, ()))
 
-  class Test extends MockHttpClient with MockAppConfig {
+        willDelete(
+          url = s"$baseUrl/income-tax/accounts/self-assessment/collection/tax-code/$nino/${taxYear.asMtd}"
```

Comment:
can we hardcode the taxYear into the url to ensure the correct non-tys endpoint is being tested